README.md

```diff
@@ -4,65 +4,36 @@
 <b>Electrode</b>
 </p>
 
-[![Gitter](https://badges.gitter.im/gitterHQ/gitter.svg)](https://gitter.im/electrode-io/electrode)
-[![Build Status][travis-image]][travis-url] [![devDependencies Status][daviddm-image]][daviddm-url]
+# Building large scale universal React Web Applications
 
-### Building large scale universal React Web Applications
+Electrode was developed by Walmart Labs in 2016 as an internal rapid application development framework for Walmart. Electrode is one of the technology engines powering the <http://www.walmart.com> eCommerce website.
 
-> Built at @WalmartLabs powering the <http://www.walmart.com> eCommerce site, Electrode is a platform for building small to large scale universal ReactJS/NodeJS web applications with a standardized structure, best practices, and modern technologies baked in. Electrode focuses on performance, component reusability, and simple deployment to multiple cloud providers—so you can focus on what makes your app unique.
+Electrode Web is a rapid application development framework; a full stack, end-to-end platform for developing, deploying, and maintaining JavaScript applications at Walmart.
 
-**It takes 5 minutes and a single command to get your own Universal React app to customize:**
+:::warning
+This framework is for the open-source version of electrode.
 
-![Hello from Electrode][hello-electrode]
+If you are a Walmart developer, please see the internal electrode X documentation site.
+:::
```

Comment:
We maintain additional code to support internal systems.  If you are a Walmart developer, please check internal docs to get started.